R/runner.R

```diff
@@ -331,6 +332,11 @@ orderly_runner_ <- R6::R6Class(
         queued <- list()
       }
       report_id <- self$con$HGET(self$keys$key_report_id, key)
+      times <- self$queue$task_times(task_id)
```

Comment:
```suggestion
      times <- round(self$queue$task_times(task_id))
```

if you want times to the nearest second